Makefile

```diff
@@ -132,7 +132,7 @@ build-bundle: ## Build the bundle image.
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 deploy-olm: operator-sdk ## deploys OLM on the cluster
-	operator-sdk olm install
+	operator-sdk olm install --version v0.18.3
```

Comment:
maybe we should use a variable for the version? so it will be more clear when we are updating it, like the OPERATOR_SDK_VERSION variable we have.

Reply:
done